test/integration/cli/cli-test.coffee

```diff
@@ -185,64 +197,57 @@ describe 'CLI', ->
         it 'should not execute any transaction', ->
           assert.deepEqual runtimeInfo.server.requestCounts, {}
 
-
       describe "and handler is killed during execution", ->
-        dreddCommandInfo = undefined
-        serverRuntimeInfo = undefined
+        runtimeInfo = undefined
 
         before (done) ->
           app = createServer()
-
           app.get '/machines', (req, res) ->
             killAll('endless-nosigterm.+[^=]foo/bar/hooks', (err) ->
               done err if err
               res.json([{type: 'bulldozer', name: 'willy'}])
             )
 
           # TCP server echoing transactions back
-          hookServer = net.createServer (socket) ->
-            socket.on 'data', (data) ->
-              socket.write data
-
-          hookServer.listen DEFAULT_HOOK_HANDLER_PORT, ->
-            server = app.listen DEFAULT_SERVER_PORT, (err, info) ->
-              serverRuntimeInfo = info
-              runDreddCommand [
-                './test/fixtures/single-get.apib'
-                "http://127.0.0.1:#{DEFAULT_SERVER_PORT}"
-                '--server=./test/fixtures/scripts/endless-nosigterm.sh'
-                '--language=./test/fixtures/scripts/endless-nosigterm.sh'
-                '--hookfiles=foo/bar/hooks'
-                '--server-wait=0'
-              ], (err, info) ->
-                dreddCommandInfo = info
-                server.close()
-
-            server.on 'close', ->
-              hookServer.close()
-              done()
+          hookHandler = net.createServer (socket) ->
+            socket.on 'data', (data) -> socket.write data
+            socket.on 'error', (err) -> console.error err
+
+          args = [
+            './test/fixtures/single-get.apib'
+            "http://127.0.0.1:#{DEFAULT_SERVER_PORT}"
+            "--server=#{COFFEE_BIN} ./test/fixtures/scripts/endless-nosigterm.coffee"
```

Comment:
It would require my deeper knowledge, so I'll rather ask - should `--server` and `--language` argument be the same?